### PR TITLE
✨Implementing a key-only view

### PR DESCRIPTION
**Pull request by FunamaYukina**
## Summary

2 changes.
Preview: https://liam-erd-sample-epqkj9r9x-route-06-core.vercel.app/

### 1st
Implementing a key-only view

ss 2549

### 2nd
Add LinkIcon and apply it as an icon for foreign keys.
This applies not only to Key Only views but also to All Fields views. 

ss 2553

### CODE REVIEW

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableColumnList/TableColumn/TableColumn.tsx [hunk 1]
```diff
@@ -35,26 +35,34 @@ export const TableColumn: FC<TableColumnProps> = ({
           className={styles.primaryKeyIcon}
           role="img"
           aria-label="Primary Key"
+          strokeWidth={1.5}
```
Comment (FunamaYukina) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableColumnList/TableColumn/TableColumn.tsx:38]: I've modified the ``strokeWidth`` to fit the design.

ss 2550

**Comment by FunamaYukina** [comment]
I'm going to refactor the code to reduce unnecessary use of map and filter functions.🙏

**Comment by FunamaYukina** [comment]
I've created a new function, ``shouldDisplayColumn``, in the TableColumnList file and refactored the code to minimize the use of map and filter functions.

commit: https://github.com/liam-hq/liam/pull/308/commits/ab664607ecfd10e62423905a0b47b19d13cf14c9
preview: https://liam-erd-sample-anywae9me-route-06-core.vercel.app/

@MH4GF 
I'd appreciate it if you could review this again🙏.

**Review by MH4GF — APPROVED** [review]
LGTM, Thanks 🚀

**Pull request merged**